nextflow_schema.json

```
@@ -80,7 +80,6 @@
                 },
                 "full_dbs": {
                     "type": "boolean",
-                    "default": false,
```

This is set to `false` by default isn't it? https://github.com/nf-core/proteinfold/blob/16fd7808df923e8fa6b352f3ced376af9e5b83cf/nextflow.config#L21
```suggestion
                    "default": false,
```